install-tools.sh

```diff
@@ -189,66 +189,53 @@ KUBECTL_1_25_8_CHECKSUM_X86_64=80e70448455f3d19c3cb49bd6ff6fc913677f4f240d368fa2
-HELM_3_6_3_CHECKSUM_X86_64=07c100849925623dc1913209cd1a30f0a9b80a5b4d6ff2153c609d11b043e262
-HELM_3_6_3_CHECKSUM_AARCH64=6fe647628bc27e7ae77d015da4d5e1c63024f673062ac7bc11453ccc55657713
 
-SOPS_3_7_3_CHECKSUM_X86_64=53aec65e45f62a769ff24b7e5384f0c82d62668dd96ed56685f649da114b4dbb
-SOPS_3_7_3_CHECKSUM_AARCH64=4945313ed0dfddba52a12ab460d750c91ead725d734039493da0285ad6c5f032
 
-LEGO_4_4_0_CHECKSUM_X86_64=302a780a56dd52601aa5d1dc31e607599cb85b113830abe464001622ca8b80a2
-LEGO_4_4_0_CHECKSUM_AARCH64=abe0e795be083143bc72ffe0f62670d96d1d33caeec2649b452d6fe9ac7ede4f
+SOPS_3_8_1_CHECKSUM_X86_64=d6bf07fb61972127c9e0d622523124c2d81caf9f7971fb123228961021811697
+SOPS_3_8_1_CHECKSUM_AARCH64=15b8e90ca80dc23125cd2925731035fdef20c749ba259df477d1dd103a06d621
 
-YQ_4_34_1_CHECKSUM_X86_64=df8b1ea3ebd84bac31691e5b77b87c798f64c845370593e56603b9892cea3e1c
-YQ_4_34_1_CHECKSUM_AARCH64=e43d788ca14c9bd949ed1c828d6073a6b42d8c78c9e454095699b1a1e844abf2
+LEGO_4_14_2_CHECKSUM_X86_64=f5a978397802a2eb20771925ceb173dff88705b45fdbb2e68312269e205fa85d
+LEGO_4_14_2_CHECKSUM_AARCH64=5050df1fb75085122cd253a3877e0d7ea07c4547964378a8f4753e1e2679cce6
 
-ORAS_1_0_0_CHECKSUM_X86_64=8533c9ea1e5a0d5eb1dfc5094c0e8ef106d15462f8a119077548f88937ed2133
-ORAS_1_0_0_CHECKSUM_AARCH64=332ba74c043ed590ab0fb61656e6d274243e915cb150d48be4ad64ed591dcc84
+YQ_4_40_5_CHECKSUM_X86_64=bccbf5ce1717ea5cec9662446b8bfa5863747ffb0a49a32e4c8dd23ada5c26fa
+YQ_4_40_5_CHECKSUM_AARCH64=e90dae67f110746a4eb7ab8bafe7362f46d1a01cb37e7db7289c30cb7a4fd13c
 
-CRANE_0_15_2_CHECKSUM_X86_64=bd5f72ae96373ac640679a6108280b6d76698773ca21f293ae30cc17413e2ad1
-CRANE_0_15_2_CHECKSUM_AARCH64=afa0bf56d95fa86fe71d5dadb5e03960b7c0586eea84270cca40b41a6d61fbb2
+ORAS_1_1_0_CHECKSUM_X86_64=e09e85323b24ccc8209a1506f142e3d481e6e809018537c6b3db979c891e6ad7
+ORAS_1_1_0_CHECKSUM_AARCH64=e450b081f67f6fda2f16b7046075c67c9a53f3fda92fd20ecc59873b10477ab4
+
+CRANE_0_18_0_CHECKSUM_X86_64=cdf4d426d965d9a8ba613d7ebf3addf93101aa2e853a3f08fbfdaed2823918f3
+CRANE_0_18_0_CHECKSUM_AARCH64=3e81dee96fb20f8ffaa150c1136abc39c376c7183843a5e0a6164b9623613c56
 
 ARGO_3_1_5_CHECKSUM_X86_64=68ebb30e79aa5ab649dbd0feb6e227b0dcff2b2983c00e176cc523a9f883567b
 ARGO_3_1_5_CHECKSUM_AARCH64=dc3c36081b6b49c8977dcffa9393a29e83568fba36a35f472caaac108674c03e
 ARGO_3_4_5_CHECKSUM_X86_64=0528ff0c0aa87a3f150376eee2f1b26e8b41eb96578c43d715c906304627d3a1
 ARGO_3_4_5_CHECKSUM_AARCH64=6d953f667ded668f351bfeb94f32e34b70badc23770c11b55e3d2bc32caa274c
 
-install_sops 3.7.3
+install_sops 3.8.1
 
-install_lego 4.4.0
+install_lego 4.14.2
 
-install_yq 4.34.1
+install_yq 4.40.5
 
-install_oras 1.0.0
+install_oras 1.1.0
 
-install_crane 0.15.2
+install_crane 0.18.0
 
-install_helm 3.6.3
-install_helm 3.7.2
-install_helm 3.8.2
 install_helm 3.9.4
 install_helm 3.10.3
 install_helm 3.11.2
+install_helm 3.12.3
+
 
-install_kubectl 1.21.3
-install_kubectl 1.22.4
-install_kubectl 1.23.17
 install_kubectl 1.24.12
 install_kubectl 1.25.8
 install_kubectl 1.26.3
```

Comment:
Upgrade to the latest 1.24.x, 1.25.x and 1.26.x releases

Add 1.27 and 1.28 latest releases